README.md

```diff
@@ -18,52 +18,106 @@ The NTT manager contracts additionally keep track of the token decimals of the o
 
 The payload includes the trimmed amount, together with the decimals that trimmed amount is expressed in. This number is the minimum of (8, source token decimals, destination token decimals).
 
-### NTT Message Lifecycle
+# Rate-Limiting and Cancel Flows
 
-### EVM
-1. **Sending**: A client calls on [`transfer`] to initiate an NTT transfer. The client must specify at minimum, the amount of the transfer, the recipient chain, and the recipient address on the recipient chain. [`transfer`] also supports a flag to specify whether the NTTManager should queue transfers if they are rate-limited. Clients can also include additional instructions to forward along to its peer NTT Manager on the recipient chain to execute. Depending on the mode, transfers are either "locked" or "burned". Once the transfer has been forwarded to the Transceiver, the NTTManager emits the following event:
+Ntt supports rate limiting on the sending. If a trnasfer is rate limited on the source chain and queueing is enabled, transfers are placed into a outbound queue that can be released after a 24-hr window via the [`completeInboundTransferQueue`] method.
+
+Transfers that are rate-limited on the destination chain are added to an InboundTransferQueue that can be released via the [`completeInboundTransferQueue`] method.
+
+If users bridge frequently between an arbitrary source chain and a given destination chain, the capacity could be exhausted quickly, leaving other users rate-limited, potentially delaying transfers. To mitigate this issue, we refilling the inbound rate limit on the source chain by the  equivalent amount of the outbound transfer. We refer to this colloquially as "Cancel Flow".
```

Comment:
If users bridge frequently between an arbitrary source chain and a given destination chain, the capacity could be exhausted quickly. This would result in other users getting rate-limited and potentially delaying transfers. To mitigate this issue, we refill the inbound rate limit on the source chain by the equivalent amount of the outbound transfer, and vice versa -- we refill the outbound rate limit on the destination chain by the equivalent amount of the inbound transfer. We refer to this colloquially as "Cancel Flow".